runtime/hydradx/src/governance/mod.rs

```diff
@@ -0,0 +1,231 @@
+//   $       $$$$$$$$$$$$$$$$$$$$$$$       writing, software distributed under the License is
+//      $$$$$$$$$$$$$$$$$$$$$$$$$$        distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+//      $$$$$$$$$   $$$$$$$$$$$         OR CONDITIONS OF ANY KIND, either express or implied.
+//        $$$$$$$$
+//          $$$$$$$$$$$$$$$$$$            See the License for the specific language governing
+//             $$$$$$$$$$$$$                   permissions and limitations under the License.
+//                $$$$$$$
+//                                                                 $$
+//  $$$$$   $$$$$                    $$                       $
+//   $$$     $$$  $$$     $$   $$$$$ $$  $$$ $$$$  $$$$$$$  $$$$  $$$    $$$$$$   $$ $$$$$$
+//   $$$     $$$   $$$   $$  $$$    $$$   $$$  $  $$     $$  $$    $$  $$     $$   $$$   $$$
+//   $$$$$$$$$$$    $$  $$   $$$     $$   $$        $$$$$$$  $$    $$  $$     $$$  $$     $$
+//   $$$     $$$     $$$$    $$$     $$   $$     $$$     $$  $$    $$   $$     $$  $$     $$
+//  $$$$$   $$$$$     $$      $$$$$$$$ $ $$$      $$$$$$$$   $$$  $$$$   $$$$$$$  $$$$   $$$$
+//                  $$$
+
+// Gov V1 (to be deprecated)
+pub mod old;
+
+pub mod origins;
+pub mod tracks;
+
+use super::*;
+use crate::governance::{
+	origins::{GeneralAdmin, ReferendumCanceller, ReferendumKiller, Spender, Treasurer, WhitelistedCaller},
+	tracks::TracksInfo,
+};
+use frame_support::{
+	parameter_types,
+	sp_runtime::Permill,
+	traits::{tokens::UnityAssetBalanceConversion, EitherOf},
+	PalletId,
+};
+use frame_system::{EnsureRoot, EnsureRootWithSuccess};
+use pallet_collective::EnsureProportionAtLeast;
+use primitives::constants::{currency::DOLLARS, time::DAYS};
+use sp_arithmetic::Perbill;
+use sp_core::ConstU32;
+use sp_runtime::traits::IdentityLookup;
+
+pub type TechCommitteeMajority = EnsureProportionAtLeast<AccountId, TechnicalCollective, 1, 2>;
+pub type TechCommitteeSuperMajority = EnsureProportionAtLeast<AccountId, TechnicalCollective, 2, 3>;
+
+parameter_types! {
+	pub const TechnicalMaxProposals: u32 = 20;
+	pub const TechnicalMaxMembers: u32 = 10;
+	pub const TechnicalMotionDuration: BlockNumber = 5 * DAYS;
+  pub MaxProposalWeight: Weight = Perbill::from_percent(50) * BlockWeights::get().max_block;
+}
+
+pub type TechnicalCollective = pallet_collective::Instance2;
+impl pallet_collective::Config<TechnicalCollective> for Runtime {
+	type RuntimeOrigin = RuntimeOrigin;
+	type Proposal = RuntimeCall;
+	type RuntimeEvent = RuntimeEvent;
+	type MotionDuration = TechnicalMotionDuration;
+	type MaxProposals = TechnicalMaxProposals;
+	type MaxMembers = TechnicalMaxMembers;
+	type DefaultVote = pallet_collective::PrimeDefaultVote;
+	type WeightInfo = weights::pallet_collective_technical_committee::HydraWeight<Runtime>;
+	type MaxProposalWeight = MaxProposalWeight;
+	type SetMembersOrigin = EitherOf<EnsureRoot<Self::AccountId>, GeneralAdmin>;
```

Comment:
this should be probably restricted to root track only

Reply:
The decision deposit for root track is extremely high, the community may see it as setting the bar for changes to TC too high.

Reply:
fair enough